spec/presenters/show_presenter_spec.rb

```diff
@@ -283,5 +283,31 @@ def mock_document_app_helper_url *args
       end
     end
   end
+
+  describe 'deprecated methods' do
+    before do
+      allow(Deprecation).to receive(:warn)
```

Comment:
I'm not sure we need to care; I wasn't setting out to test if `deprecation` was working, just to keep it quiet.